File: program_data/github_java_sort_function/6/100.java
import java.util.ArrayList;
import java.util.LinkedList;
import java.util.List;
import java.util.Queue;
import java.util.Scanner;

public class RadixSort {

    private static void radixSort(int[] arr, int length) {
        int max = getMax(arr, length);

        for (int exp = 1; max / exp > 0; exp *= 10)
            sort(arr, length, exp);
    }

    private static void sort(int[] arr, int length, int exp) {
        List<Queue<Integer>> q = new ArrayList<Queue<Integer>>();
        for (int i = 0; i < 10; i++)
            q.add(i, new LinkedList<>());

        for (int i = 0; i <= length; i++) {
            q.get((arr[i] / exp) % 10).add(arr[i]);
        }

        int i = 0, k = 0;
        while (i < 10) {
            while (!q.get(i).isEmpty()) {
                arr[k++] = q.get(i).remove();
            }
            i++;
        }
    }

    private static int getMax(int[] arr, int length) {
        int mx = arr[0];
        for (int i = 1; i < length; i++)
            if (arr[i] > mx)
                mx = arr[i];
        return mx;
    }
}

File: program_data/PKU_raw/75/100.c
//********************************
//*????????             **
//*??????                 **
//*???2012.11.12             **
//*******************************/

int main ()
{
	int a[1000], b[1000], t[1001] = {0};
	int n = 0, i, max = 0;
	do
	{
		cin >> a[n];
		n++;

	}while (cin.get() != '\n');
	for (i = 0; i < n; i++)
	{
		cin >> b[i];
		for ( ; a[i] < b[i]; a[i]++)
			t[a[i]]++;
		cin.get();
	}
	for (i = 0; i < 1000; i++)
	{
		if (t[i] > max)
			max = t[i];
	}
	printf("%d %d", n, max);
	return 0;
}
	
